src/QMCWaveFunctions/BsplineFactory/createComplexDouble.cpp

```diff
@@ -36,10 +37,24 @@ BsplineReaderBase* createBsplineComplexDouble(EinsplineSetBuilder* e, bool hybri
   BsplineReaderBase* aReader = nullptr;
 
 #if defined(QMC_COMPLEX)
-  if (hybrid_rep)
-    aReader = new HybridRepSetReader<HybridRepCplx<SplineC2C<double>>>(e);
+#if defined(ENABLE_OFFLOAD)
+  if (useGPU == "yes")
+  {
+    if (hybrid_rep)
+    {
+      APP_ABORT("OpenMP offload has not been enabled with hybrid orbital representation!");
```

Comment:
The new code must always work and never abort if offload code is not present. The CPU path should be used when hybrid rep is selected for now. Same goes for real splines. Create an issue if you don't want to fix this now.

Reply:
No reason to need more user actions. Fallback to host code path.